doc/release/upcoming_changes/14510.compatibility.rst

```diff
@@ -8,3 +8,5 @@ with::
     dropped_arr = drop_fields(arr, ['a', 'b'])
     if dropped_arr.dtype.names == ():
         dropped_arr = None
+
+.. comment to force a blank line
```

Comment:
towncrier appends a nicely formatted link to the PR, but does not leave a blank line. However the code block requires a blank line after it. Perhaps towncrier should be using more vertical space?

Reply:
the comment was not sufficient, added some redundant text to force a blank line

Reply:
We can add  a blank line to the template.rst

Reply:
Where?

Reply:
```diff
diff --git a/doc/release/upcoming_changes/template.rst b/doc/release/upcoming_changes/template.rst
index 9c8a3b5fc..997b4850e 100644
--- a/doc/release/upcoming_changes/template.rst
+++ b/doc/release/upcoming_changes/template.rst
@@ -17,6 +17,7 @@
 {% if definitions[category]['showcontent'] %}
 {% for text, values in sections[section][category].items() %}
 {{ text }}
+
 {{ get_indent(text) }}({{values|join(', ') }})
 
 {% endfor %}
```
But that will do that even for bullet points, which I guess should work? Maybe have to try and see if it isn't ugly.